cmd/tracker-swebench/analyze.go

```
@@ -0,0 +1,1061 @@
+
+// findEvaluatorReport scans resultsDir for a file that looks like an SWE-bench
+// evaluator report and extracts its `resolved_ids`. Returns (nil, "", nil) if
+// no such file is present — that's the expected degraded mode. The report
+// file conventions we understand:
+//   - any top-level JSON object with a "resolved_ids" array
+//   - file names: *.evaluation.json, *.report.json, resolved_ids.json, resolution.json
+func findEvaluatorReport(resultsDir string) (map[string]struct{}, string, error) {
+	candidates := []string{}
+	entries, err := os.ReadDir(resultsDir)
+	if err != nil {
+		return nil, "", nil // dir exists (we stat'd it), so this is rare; don't fail the report
```

golangci-lint (2.11.4)

[error] error is not nil (line 542) but it returns nil

(nilerr)